validation/src/main/scala/hmda/validation/rules/lar/validity/V338.scala

```diff
@@ -0,0 +1,29 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V338 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V338"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when(notANaturalPerson(lar)) {
+      lar.applicant.income is equalTo("NA")
+    }
+  }
+
+  private def notANaturalPerson(lar: LoanApplicationRegister): Result = {
+    val okAction = List(1, 2, 3, 4, 5, 7, 8)
+
+    (lar.applicant.ethnicity is equalTo(4)) and
```

Comment:
Should I put these values into a List and compare it with `List(4,7,4,5,8,5)`?  I wasn't sure if that would be a better approach.